src/Microsoft.ML.Transforms/Text/TextStaticExtensions.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data.StaticPipe.Runtime;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.Transforms.Text
+{
+    /// <summary>
+    /// Extensions for statically typed word tokenizer.
+    /// </summary>
+    public static class WordTokenizerExtensions
+    {
+        private sealed class OutPipelineColumn : VarVector<string>
+        {
+            public readonly Scalar<string> Input;
+
+            public OutPipelineColumn(Scalar<string> input, string separators)
+                : base(new Reconciler(separators), input)
+            {
+                Input = input;
+            }
+        }
+
+        private sealed class Reconciler : EstimatorReconciler
+        {
+            private readonly string _separators;
+
+            public Reconciler(string separators)
+            {
+                _separators = separators;
+            }
+
+            public override IEstimator<ITransformer> Reconcile(IHostEnvironment env,
+                PipelineColumn[] toOutput,
+                IReadOnlyDictionary<PipelineColumn, string> inputNames,
+                IReadOnlyDictionary<PipelineColumn, string> outputNames,
+                IReadOnlyCollection<string> usedNames)
+            {
+                Contracts.Assert(toOutput.Length == 1);
+
+                var pairs = new List<(string input, string output)>();
+                foreach (var outCol in toOutput)
+                    pairs.Add((inputNames[((OutPipelineColumn)outCol).Input], outputNames[outCol]));
+
+                return new WordTokenizer(env, pairs.ToArray(), _separators);
+            }
+        }
+
+        /// <summary>
+        /// Tokenize incoming text using <paramref name="separators"/> and output the tokens.
+        /// </summary>
+        /// <param name="input">The column to apply to.</param>
+        /// <param name="separators">The separators to use (comma separated).</param>
+        public static VarVector<string> TokenizeText(this Scalar<string> input, string separators = "space") => new OutPipelineColumn(input, separators);
```

Comment:
>"space" [](start = 100, length = 7)

I think we should make them actual space(' ') or tab ('\t') on the front facing API. What do you think?

Reply:
I think it's better to just do #935 and not have some form of half-solutions here. I am ok with replacing `"space"` with `" "`, but it just begs the question: why not `' '`, and that's a tougher change.

---
In reply to: [218548057](https://github.com/dotnet/machinelearning/pull/931#discussion_r218548057) [](ancestors = 218548057)